bitnami/etcd/README.md

```diff
@@ -0,0 +1,178 @@
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release bitnami/etcd
+```
+
+The command deploys etcd on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the etcd chart and their default values.
+
+|          Parameter                 |                                                  Description                                               |                                     Default                        |
+|------------------------------------|------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------- |
+| `image.registry`                   | etcd image registry                                                                                        | `docker.io`                                                        |
+| `image.repository`                 | etcd Image name                                                                                            | `bitnami/etcd`                                                     |
+| `image.tag`                        | etcd Image tag                                                                                             | `{VERSION}`                                                        |
+| `image.pullPolicy`                 | etcd image pull policy                                                                                     | `Always`                                                           |
+| `image.pullSecrets`                | Specify image pull secrets                                                                                 | `nil` (does not add image pull secrets to deployed pods)           |
+| `image.debug`                      | Specify if debug values should be set                                                                      | `false`                                                            |
+| `updateStrategy`                   | Update strategy for the stateful set                                                                       | `RollingUpdate`                                                    |
```

Comment:
Add **rollingUpdatePartition** parameter to configure the partition update strategy as we did on other charts such as Redis.

Reply:
done